packages/next/src/server/request/params.ts

```diff
@@ -103,7 +103,7 @@ export function createPrerenderParamsForClientSegment(
           // This params object has one of more fallback params so we need to consider
           // the awaiting of this params object "dynamic". Since we are in dynamicIO mode
           // we encode this as a promise that never resolves
-          return makeHangingPromise()
+          return makeHangingPromise(prerenderStore.renderSignal, '`params`')
```

Comment:
nit: in other places, we add the backticks in the error message itself instead of passing them in, maybe let's be consistent with that?

Reply:
Yeah I want to move to this style, just haven’t updated all the old locations. Sometimes the expression is more than one thing or just descriptive so having it be in the passed in argument is better than having it assume the passed in argument is literal code